lms/djangoapps/teams/static/teams/js/views/edit_team_members.js

```diff
@@ -0,0 +1,103 @@
+                    this.teamMembershipDetailUrl = this.teamMembershipDetailUrl.substring(
+                        0, this.teamMembershipDetailUrl.lastIndexOf('team_id')
+                    ) + this.model.get('id') + ",";
+
+                    this.teamEvents = options.teamEvents;
+                },
+
+                render: function() {
+                    if (this.model.get('membership').length === 0) {
+                        this.$el.html('<p>' + gettext('This team does not have any members.') + '</p>');
+                    }
+                    else {
+                        this.$el.html('<ul class="edit-members"></ul>');
+                        this.renderTeamMembers();
+                    }
+                    return this;
+                },
+
+                renderTeamMembers: function() {
+                    var self = this, dateJoined, lastActivity;
+
+                    _.each(this.model.get('membership'), function(membership) {
+                        dateJoined = interpolate(
+                            // Translators: 'date' is a placeholder for a fuzzy, relative timestamp (see: https://github.com/rmm5t/jquery-timeago)
+                            gettext("Joined %(date)s"),
+                            {date: self.dateTemplate({date: membership.date_joined})},
+                            true
+                        );
+
+                        lastActivity = interpolate(
+                            // Translators: 'date' is a placeholder for a fuzzy, relative timestamp (see: https://github.com/rmm5t/jquery-timeago)
+                            gettext("Last Activity %(date)s"),
+                            {date: self.dateTemplate({date: membership.last_activity_at})},
+                            true
+                        );
+
+                        // It is assumed that the team member array is automatically in the order of date joined.
+                        self.$('.edit-members').append(self.teamMemberTemplate({
+                            imageUrl: membership.user.profile_image.image_url_medium,
+                            username: membership.user.username,
+                            memberProfileUrl: '/u/' + membership.user.username,
+                            dateJoined: dateJoined,
+                            lastActive: lastActivity
+                        }));
+                    });
+                    this.$('abbr').timeago();
+                },
+
+                removeMember: function (event) {
+                    var self = this, username = $(event.currentTarget).data('username');
+                    event.preventDefault();
+
+                    ViewUtils.confirmThenRunOperation(
+                        gettext('Remove this team member?'),
+                        gettext('The member will be removed from the team, allowing another user to take the available spot.'),
+                        gettext('Remove'),
```

Comment:
@cahrens Is "Remove" the button label? In the current sandbox I'm seeing "Delete" and "Close" as the button labels in the confirmation dialog.